### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by NicksonYap**
Remind to add provider into `config/app.php`
And an IndexConfigurator is required for each Model

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,11 +52,24 @@ The package has been tested in the following configuration:
 Use composer to install the package:
 
 ```
+composer require laravel/scout
```
Comment (babenkoivan) [inline README.md]: Actually, `scout-elasticsearch-driver` requires `laravel/scout` in its composer.json, so no need to require it manually.

File: README.md [hunk 2]
```diff
@@ -52,11 +52,24 @@ The package has been tested in the following configuration:
...
+```
+'providers' => [
+
+         ...
+         
+        //Laravel Scout & Elasticsearch
+        Laravel\Scout\ScoutServiceProvider::class,
+        ScoutElastic\ScoutElasticServiceProvider::class,
+    ],
+```
```
Comment (babenkoivan) [inline README.md]: This package supports [Laravel package discovery](https://laravel.com/docs/5.6/packages#package-discovery), so no need to manually add providers to `app.php`. It makes sense though if you are using old Laravel version, but in the requirements I have 5.6 version.

Comment (NicksonYap) [reply README.md]: Regarding composer require:
https://laravel.com/docs/5.6/scout#installation
This is how we were told to install and it works even after `git pull` from origin (on my production server) and running `composer install`
It should automatically be added into `composer.json`

Yes, I came from 5.4
But it should be there for Scout to work so it'll be a good to remind anyways
You can specify that it is already added for 5.6

**Comment by babenkoivan** [comment]
Thank you for the contribution, I updated README in v3.8.1.

**Pull request closed without merging**